site/docs/how-to/review.md

```diff
@@ -0,0 +1,79 @@
+
+We set forth the best convention to review infrastructure changes within transactions which:
+- works with any cloud identity and access management structure
+- works with IaSQL schema upgrades
+- works with any version control system, not just GitHub, without the need for privileged access
+- preserves the ability to `iasql_preview` changes in development and get the type-safety feedback from the dashboard editor
+
+```sql title="Create review for an infrastructure change within a transaction"
+SELECT * FROM iasql_begin();
+
+UPDATE log_group SET log_group_name = 'test' WHERE id = 4;
+
+SELECT * FROM iasql_create_review('My review' )
+```
+
+The raw markdown content returned will look as follows:
+
+```
+# My review
+
+Review description
+
+## IaSQL Preview
+
+| action | table_name | id | description |
+| --- | --- | --- | --- |
+| update | log_group | 4 | quickstart-log-group|us-east-2 |
+
+## SQL changes
+
+```sql
+
+UPDATE log_group
+SET log_group_name = 'quickstart-log-group', log_group_arn = 'test', creation_time = '2023-01-25T11:17:14.045+00:00', region = (SELECT region FROM aws_regions WHERE region = 'us-east-2')
+WHERE log_group_name = 'quickstart-log-group' AND log_group_arn = 'arn:aws:logs:us-east-2:257682470237:log-group:quickstart-log-group:*' AND creation_time = '2023-01-25T11:17:14.045+00:00' AND region = (SELECT region FROM aws_regions WHERE region = 'us-east-2');
+
+```
+
+The above string should be pasted into a markdown file within a version-controlled repository of your choosing and submitted for review by the relevant team members. Once the change has been approved and merged, come back to the transaction in the SQL REPL and simply commit it.
+
+<!-- TODO allow passing an optional message to IaSQL commit which can be the URL of the PR -->
+
+```sql title="Commit transaction once reviews has been accepted"
+SELECT * FROM iasql_commit();
+```
+
+Below is the visualization in markdown of the above string:
+
+
+----
+
+
+# Change log group name
+
+Review description
+
+## IaSQL Preview
+
+| action | table_name | id | description |
+| --- | --- | --- | --- |
+| update | log_group | 4 | quickstart-log-group|us-east-2 |
+
+## SQL changes
+
+```sql
+
+UPDATE log_group
+SET log_group_name = 'quickstart-log-group', log_group_arn = 'test', creation_time = '2023-01-25T11:17:14.045+00:00', region = (SELECT region FROM aws_regions WHERE region = 'us-east-2')
+WHERE log_group_name = 'quickstart-log-group' AND log_group_arn = 'arn:aws:logs:us-east-2:257682470237:log-group:quickstart-log-group:*' AND creation_time = '2023-01-25T11:17:14.045+00:00' AND region = (SELECT region FROM aws_regions WHERE region = 'us-east-2');
+
+```
```

Comment:
I would move this up to before the `iasql_commit` call, perhaps saying something like "if you put this into a Github PR or other tool that pretty-prints markdown, it will look like this"

Reply:
I disagree. Rather, I did that initially and it becomes really hard to follow what is the example versus the actual doc so I would like to keep it here